superset-frontend/src/SqlLab/components/ResultSet.tsx

```diff
@@ -166,14 +210,148 @@ export default class ResultSet extends React.PureComponent<
     }
   }
 
+  handleOverwriteDatasetOption(data, option) {
+    this.setState({ datasetToOverwrite: option })
+  }
+
+  handleOverwriteDataset() {
+    const { sql, results } = this.props.query;
+    const { datasetToOverwrite } = this.state;
+
+    SupersetClient.put({
+      endpoint: `api/v1/dataset/${datasetToOverwrite.dataSetId}?override_column=true`,
+      headers: { 'Content-Type': 'application/json' },
```

Comment:
is this content type the default?

Reply:
Not sure, but copied some `supersetClient` network calls and they had the headers set the same way

Reply:
It is, you can see where this is handled in the backend: https://github.com/apache/incubator-superset/blob/master/superset/datasets/api.py#L254-L255